.github/workflows/package.yml

```diff
@@ -12,7 +12,27 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v4
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
+      - name: Build
+        run: |
+          pixi run build
+      - uses: actions/upload-artifact@v4
         with:
-          ref: ${{ github.head_ref }}
-          fetch-depth: 0
-      # TODO: Build and upload to PyPI.
+          name: artifact
+          path: dist/*
+
+  upload_pypi:
+    name: Upload to PyPI
+    needs: [build]
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: artifact
+          path: dist
+      - uses: pypa/gh-action-pypi-publish@v1.9.0
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
```suggestion
```

you can use OIDC https://github.com/Quantco/multiregex/pull/52#issuecomment-2127001861